### PR TITLE
README cleanup

### PR DESCRIPTION
**Pull request by askmeegs**

### CODE REVIEW

**Comment by murog** [comment]
re-running ci

**Comment by murog** [comment]
contacts test failure with CloudTraceSpanExporter
```
plugins: cov-2.10.0
collecting ... collected 4 items / 1 error / 3 selected

==================================== ERRORS ====================================
___________________ ERROR collecting tests/test_contacts.py ____________________
tests/test_contacts.py:27: in <module>
    from contacts.contacts import create_app
contacts.py:30: in <module>
    from opentelemetry.exporter.cloud_trace import CloudTraceSpanExporter
/home/sanche/venv-contacts/lib/python3.5/site-packages/opentelemetry/exporter/cloud_trace/__init__.py:302: in <module>
    trace_api.SpanKind.INTERNAL: ProtoSpan.SpanKind.INTERNAL,
E   AttributeError: SpanKind
=========================== short test summary info ============================
ERROR tests/test_contacts.py - AttributeError: SpanKind
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
=============================== 1 error in 1.08s ===============================
##[error]Process completed with exit code 2.
```

File: README.md [hunk 1]
```diff
@@ -1,15 +1,25 @@
...
-Bank of Anthos allows users to create artificial accounts and simulate transactions between accounts.
-Bank of Anthos was developed to create an end-to-end sample demonstrating Anthos best practices.
+**Bank of Anthos** is a sample HTTP-based web app that simulates a bank's payment processing network, allowing users to create artificial bank accounts and complete transactions. 
 
-## Architecture
+Google uses this application to demonstrate how developers can modernize enterprise applications using GCP products, including: [GKE](https://cloud.google.com/kubernetes-engine), [Anthos Service Mesh](https://cloud.google.com/anthos/service-mesh), [Anthos Config Management](https://cloud.google.com/anthos/config-management), [Migrate for Anthos](https://cloud.google.com/migrate/anthos), [Spring Cloud GCP](https://spring.io/projects/spring-cloud-gcp), and [Cloud Operations](https://cloud.google.com/products/operations). This application works on any Kubernetes cluster. 
 
-![Architecture Diagram](./docs/architecture.png)
+If you’re using this app, please ★Star the repository to show your interest!
```
Comment (murog) [inline README.md:9]: 🌟

File: docs/development.md [hunk 2]
```diff
@@ -57,6 +57,16 @@ If you're adding a new feature to one or more of the Java services (`ledgerwrite
 2. Re-run `skaffold dev` or `skaffold run` to trigger a Jib container build using Maven and the updated pom file. 
 
 
+## Generating your own JWT public key.
```
Comment (murog) [inline docs/development.md:60]: Can the app run without the JWT key currently?  I've only used the skaffold file which creates the JWT secret

File: README.md [hunk 3]
```diff
@@ -102,87 +104,22 @@ EXTERNAL-IP
...
-```
-make monolith
-```
+## Development 
 
-## Local Development
+See the [Development guide](docs/development.md) to learn how to run and develop this app locally. 
 
-See the [Development Guide](./docs/development.md) for instructions on how to build and develop services locally, and the [Contributing Guide](./CONTRIBUTING.md) for pull request and code review guidelines.
+## Talks/Demos using Bank of Anthos
```
Comment (murog) [inline README.md:122]: great idea!

File: .github/workflows/ci-pr.yaml [hunk 4]
```diff
@@ -70,7 +71,7 @@ jobs:
           echo "testing $SERVICE..."
           # save current working dir to memory and cd to src/$SERVICE
           pushd src/$SERVICE
-            python3 -m venv $HOME/venv-$SERVICE
+            python3 -m venv --clear $HOME/venv-$SERVICE
```
Comment (murog) [inline .github/workflows/ci-pr.yaml:74]: created issue for this, i think best to keep in separate pr's

**Review by murog — APPROVED** [review]
Thanks for the updates! Added a couple comments but overall LGTM

File: .github/workflows/ci-pr.yaml (hunk 4 above)
Comment (askmeegs) [reply .github/workflows/ci-pr.yaml]: removed!

File: docs/development.md (hunk 2 above)
Comment (askmeegs) [reply docs/development.md:60]: I think the deployments depend on the JWT secret to start up -but I actually think the app might be able to run without it? not sure.
